src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -926,7 +928,7 @@ class AddItemsPage extends Component {
 
             this.setState({
               values: { ...this.state.values, lineItems: lineItemsAfterSave },
-              currentLineItems: lineItemsAfterSave,
+              currentLineItems: lineItemsBackendData,
```

Comment:
Why `lineItems` are `lineItemsAfterSave`, and `currentLineItems` are `lineItemsBackendData`? Should these two be different in this state?

Reply:
It is connected to the fact, that lines with 0 qty are not saved in database, but those lines should be visible in the table. Setting `currentLineItems` to `lineItemsBackendData`resolves the problem with duplicated items, because if I tried to change the quantity of items fastly to 0, some of the items were not deleted